practice-one/src/styles/sections/service.css

```diff
@@ -50,3 +50,27 @@
 .service-button {
   margin-top: 50px;
 }
+
+.service:nth-child(1)::after {
+  content: 'Online order';
```

Comment:
Wrong content. Because in services list, content is not fixable. So we should use it as pseudo. If so, your UI will broken if text is too long

Reply:
I has just researched more about this content,  ::after is primarily for adding non-essential, decorative content ,thanks for your contribution.